coralproject/talk
# Reader flags

## What does this PR do?

Adds flagging functionality to talk.

## How do I test this PR?

1. `npm build` and `npm start`
2. Navigate to `http://localhost:3000/admin/embed/stream/preview`
3. Post a comment (if one does not exist) and click "Flag"
4. Button text should change to "Flagged" and icon should become red.
5. Check network tab: a successful post request should have been made to `http://localhost:3000/api/v1/comments/some-comment-id/actions`
6. Reload the page. The flags should be cleared.
7. Check the redux store. The comment you flagged should have a `flag` link and the store should have an `actions` object with an action of the proper format inside.

## Notes

- As per discussions with the team, I have split the call from `stream` and the redux store into separate action types.
- Tests for `coral-framework` have been moved into the main test folder and brought up to date.
- As of a recent merge with master, linting has been improved and lint errors have resolved across `coral-embed-stream` and `coral-admin`.

@coralproject/tech
